docs/tables/github_code_owner.md

```diff
@@ -0,0 +1,52 @@
+# Table: github_code_owner
+
+All of your code owners rules defined in your repository, in CODEOWNERS files.
+
+The `github_code_owner` table can be used to query information about **ANY** repository, and **you must specify which repository** in the where or join clause (`where repository_full_name=`, `join github_code_owner on repository_full_name=`).
+
+## Examples
+
+### Get All your CodeOwners rules about a specific repository
+
+```sql
+select
+  line,
+  repository_full_name,
+  pattern,
+  users,
+  teams,
+  pre_comments,
+  line_comment 
+from
+  github.github_code_owner 
+where
+  repository_full_name = 'my-orga/code-owners' 
```

Comment:
```suggestion
  repository_full_name = 'github/docs' 
```

Reply:
I think linking to a real example instead of having users search out repos with a CODEOWNERS file will make it easier for users to use this table and understand's returned